### ISSUE TITLE
[core] [pages.class.php] $dbFields: Switch to protected

### ISSUE THREAD

**Issue by G3N-es** [bludit/bludit#919]
### Describe your problem
I want to expand the fields in child classes to create new types of content.
Private $dbFields make it difficult.

https://github.com/bludit/bludit/blob/master/bl-kernel/pages.class.php

Solution:
```
class Pages extends dbJSON {
	private $parentKeyList = array();
	/*private*/ protected $dbFields = array(
```

**Comment by dignajar** [comment]
I will change

**Comment by G3N-es** [comment]
Thanks :-)

**Comment by dignajar** [comment]
This issue was implemented in 3.7.1. thanks
